x-pack/plugins/ml/public/components/field_title_bar/field_title_bar.js

```diff
@@ -18,24 +19,32 @@ export function FieldTitleBar({ card }) {
     return null;
   }
 
+  const fieldName = card.fieldName || i18n.translate('xpack.ml.fieldTitleBar.documentCountLabel', {
+    defaultMessage: 'document count'
+  });
+  const cardTitleAriaLabel = [fieldName];
+
   const classNames = ['ml-field-title-bar'];
   if (card.fieldName === undefined) {
     classNames.push('document_count');
   } else if (card.isUnsupportedType === true) {
     classNames.push('type-other');
+    cardTitleAriaLabel.unshift(
```

Comment:
This is only adding the field type to the aria label if the type is an unsupported type. For all other types, the aria label does not include the field type. I think the handling of the `classNames` and the `cardTitleAriaLabel` should be kept separate, something like

```
  if (card.fieldName === undefined) {
    classNames.push('document_count');
  } else if (card.isUnsupportedType === true) {
    classNames.push('type-other');
  } else {
    classNames.push(card.type);
  }

  if (!card.isUnsupportedType) {
    cardTitleAriaLabel.unshift(getMLJobTypeAriaLabel(card.type));
  }
```